src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +194,58 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await ImageRow.from_alias(session, alias)
+            if image_row is None:
+                raise UnknownImageReference
+
+            for key, value in validate_image_labels(image_row.labels).items():
+                if key[:11] == "ai.backend.":
+                    print(f"{key[11:]:<30}: ", end="")
+                else:
+                    print(f"{key:<30}: ", end="")
+                print(*value) if isinstance(value, list) else print(value)
+
+        except UnknownImageReference:
+            log.error(f"No images were found with alias: {alias}")
+        except Exception:
+            log.exception("An error occurred.")
+
+
+async def validate_image_canonical(
+    cli_ctx, canonical: str, architecture: Optional[str] = None
+) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            # Default arch is current arch
+            if architecture is None:
+                architecture = CURRENT_ARCH
```

Comment:
How about fetching image-row without `architecture` condition if it is `None`? 
Then the result of DB fetch can be multiple image rows and we just print them all

Reply:
Joong-gi said that the basic architecture would be better to follow the architecture of the user machine currently using, so I made it like this!
How about add click.option for all architecture's image info?

Reply:
I see. this is good enough for me!

Reply:
@fregataa I've added a  click.option  Please check it out :)